examples/batching_strategies/volume_batching/src/volume_batching.cc

```diff
@@ -108,12 +108,42 @@ TRITONBACKEND_ModelBatchIncludeRequest(
 /// \param model The backend model for which Triton is forming a batch.
 /// \param userp The placeholder for backend to store and retrieve information
 /// about this pending batch.
+/// \param cache_userp The read-only placeholder for backend to retrieve
+// information about the batching strategy for this model.
 /// \return a TRITONSERVER_Error indicating success or failure.
 TRITONSERVER_Error*
-TRITONBACKEND_ModelBatchInitialize(TRITONBACKEND_Model* model, void** userp)
+TRITONBACKEND_ModelBatchInitialize(
+    TRITONBACKEND_Model* model, void** userp, const void* cache_userp)
 {
   // Userp will point to an unsigned integer representing the remaining volume
   // in bytes for this batch.
+  *userp = new unsigned int(*static_cast<const unsigned int*>(cache_userp));
+  return nullptr;  // success
+}
+
+/// Callback to be invoked when Triton has finished forming a batch.
+/// \param userp The placeholder for backend to store and retrieve information
+/// about this pending batch.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_Error*
+TRITONBACKEND_ModelBatchFinalize(void* userp)
+{
+  delete static_cast<unsigned int*>(userp);
+  return nullptr;  // success
+}
+
+/// Callback to be invoked when Triton loads model.
+/// This will hold a cached user pointer that can be read during custom
+/// batching. \param model The backend model for which Triton is forming a
```

Comment:
NIT: Put the `\param`s and `\return` on a new line

Reply:
Thanks for the feedback! Updated the comments.